packages/main/src/ColorPalette.ts

```diff
@@ -432,9 +512,34 @@ class ColorPalette extends UI5Element {
 	_onDefaultColorClick() {
 		if (this.defaultColor) {
 			this._setColor(this.defaultColor);
+			const defaultColorItem = this.defaultColorItem;
+			this._shouldFocusRecentColors = true;
+
+			if (!this._recentColors.includes(this.defaultColor) && this.showRecentColors) {
+				this._recentColors.unshift(this.defaultColor);
```

Comment:
Can we take this add to recent colors logic into a method and use it when we add to recent colors? It will be like a helper funciton